internal/pkg/util/starter/starter.go

```diff
@@ -68,6 +68,16 @@ func UseSuid(suid bool) CommandOp {
 	}
 }
 
+// IsSuidInstall returns true if the privileged binary is configured and exists
+func IsSuidInstall() bool {
+	if buildcfg.APPTAINER_SUID_INSTALL != 1 {
+		return false
+	}
+	path := filepath.Join(buildcfg.LIBEXECDIR, "apptainer/bin/starter-suid")
+	_, err := os.Stat(path)
+	return err == nil
```

Comment:
This could be confusing in case the setuid bit is dropped for some reason but the file exist, I think we can just return true without checking its presence and let it fail later with explicit error messages